common/kvstore/tablestore/table_store_builder.go

```diff
@@ -0,0 +1,374 @@
+package tablestore
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"github.com/Layr-Labs/eigenda/common/kvstore"
+	"github.com/Layr-Labs/eigenda/common/kvstore/leveldb"
+	"github.com/Layr-Labs/eigenda/common/kvstore/mapstore"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"math"
+	"sort"
+)
+
+// The table ID reserved for the metadata table.
```

Comment:
Comment what's in the metadata table (like what you did for namespaceTable)

Reply:
```
// The table ID reserved for the metadata table. The metadata table is used for internal bookkeeping.
// The following data is currently stored in the metadata table:
// - Schema version (in case we ever need to do a schema migration)
// - Table deletion markers used to detect a crash during table deletion and to complete 
//   the deletion when the store is next started.
```